src/libos/src/fs/file_ops/fallocate.rs

```diff
@@ -0,0 +1,11 @@
+use super::*;
+
+pub fn do_posix_fallocate(fd: FileDesc, offset: u64, len: u64) -> Result<()> {
```

Comment:
Since the File trait provides `fallocate` (with the `mode` argument), not `posix_fallocate`, I think we should implement `fallocate` rather than `posix_fallocate`.

Reply:
OK